<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="TeInfoGBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>821</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어_ac</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Confidence</string>
   </property>
   <widget class="QGroupBox" name="testGBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>문제지</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>19</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>방금 푼 문제는 어느 정도 확신을 가지고 정답을 선택하셨나요?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cnfGBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>691</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>선택지</string>
    </property>
    <widget class="QRadioButton" name="cnfRBtn1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cnfRBtn2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cnfRBtn3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cnfRBtn4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cnfRBtn5">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="btn1Label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>확신없음</string>
     </property>
    </widget>
    <widget class="QLabel" name="btn5Label">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매우확신</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cnfRBtn6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cnfRBtn7">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="cnfCntLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>1 / 6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cnfSubmitBtn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>제출하기</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="etcGBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Etc.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
